contentful_lite.gemspec

```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |spec|
   spec.name          = "contentful_lite"
   spec.version       = ContentfulLite::VERSION
   spec.authors       = ["Gonzalo Munoz", "Daniel Lopez"]
-  spec.email         = ["gonzalo.munoz@juul.com", "dlopez@juul.com"]
+  spec.email         = ["hq063gon@gmail.com", "dlopez@juul.com"]
```

Comment:
Could you please also add myself and/or Er-Dev@juul.com so that there’s a current JUUL contact as well.